module/TaskManagement/src/TaskManagement/Service/NotifyMailListener.php

```diff
@@ -77,6 +95,21 @@ public function processTaskClosed(Event $event){
 		$task = $this->taskService->findTask($taskId);
 		$this->sendTaskClosedInfoMail($task);
 	}
+	
+	public function processWorkItemIdeaCreated(Event $event) {
+		$streamEvent = $event->getTarget ();
+		$taskId = $streamEvent->metadata ()['aggregate_id'];
+		$task = $this->taskService->findTask ( $taskId );
+		
+		if ($task->getStatus() == Task::STATUS_IDEA) {
+			$memberId = $event->getParam ( 'by' );
+			$member = $this->userService->findUser ( $memberId );
```

Comment:
Devo usare per forza lo `userService` per recuperere l'`user`, infatti `$task->getMember($memberId)->getUser()` non è a disposizione nell'entità `Task` restituita dal `taskService`. Si può riscontrare questo anche dagli altri metodi realizzati in precedenza che utilizzano la stessa modalità di "recupero" dello `User`.